evalml/tests/dependency_update_check/minimum_core_requirements.txt

```diff
@@ -1,18 +1,19 @@
-numpy==1.19.1
-pandas==1.1.0
-scipy==1.2.1
-scikit-learn==0.23.1
+numpy==1.20.0
+pandas==1.2.1
+scipy==1.3.3
+scikit-learn==0.24.0
 scikit-optimize==0.8.1
 pyzmq==20.0.0
 colorama==0.4.4
-cloudpickle==0.2.2
-click==7.0.0
+cloudpickle==1.5.0
+click==7.1.2
 psutil==5.6.3
 requirements-parser==0.2.0
 shap==0.36.0
 texttable==1.6.2
 woodwork==0.0.11
-featuretools==0.20.0
+featuretools==0.21.0
 dask==2.12.0
+distributed==2.12.0
```

Comment:
How come we need to add this package? Should be pulled in by featuretools no?

Reply:
The reason I had to add this is the following pip warning/error (which I saw after installing `minimum_test_requirements.txt`):
```
distributed 2021.5.0 has requirement dask==2021.05.0, but you'll have dask 2.12.0 which is incompatible.
```